packages/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -17,83 +17,89 @@ const underlinedItem = (color: string): ICSSInJSStyle => ({
 })
 
 const getActionStyles = ({
-  props: { primary, underlined, iconOnly },
+  props: { primary, underlined, iconOnly, color },
   variables: v,
-  color,
+  colorScheme,
 }: {
   props: MenuItemPropsAndState
   variables: MenuVariables
-  color: string
+  colorScheme: ColorScheme
 }): ICSSInJSStyle =>
   underlined || iconOnly
     ? {
-        color,
+        color: v.color,
       }
-    : primary
+    : primary || color
     ? {
-        color: v.primaryActiveColor,
-        background: v.primaryActiveBackgroundColor,
+        color: colorScheme.foregroundActive,
+        background: v.backgroundColorActive || colorScheme.backgroundActive,
       }
     : {
-        color,
-        background: v.activeBackgroundColor,
+        color: v.color,
+        background: v.backgroundColorActive || colorScheme.backgroundActive,
       }
 
 const getFocusedStyles = ({
   props,
   variables: v,
-  color,
+  colorScheme,
 }: {
   props: MenuItemPropsAndState
   variables: MenuVariables
-  color: string
+  colorScheme: ColorScheme
 }): ICSSInJSStyle => {
-  const { primary, underlined, isFromKeyboard, active, vertical } = props
+  const { primary, color, underlined, isFromKeyboard, active, vertical } = props
   if (active && !underlined && !vertical) return {}
   return {
-    ...(underlined && !isFromKeyboard
-      ? {
-          color,
-        }
-      : primary
-      ? {
-          color: v.primaryFocusedColor,
-          background: v.primaryFocusedBackgroundColor,
-        }
-      : {
-          color,
-          background: v.hoverBackgroundColor,
-        }),
-
+    color: primary || color ? colorScheme.foregroundFocus : v.colorActive,
```

Comment:
I see this kind of logic throughout the whole file, but sometimes `color` props as priority, other times a variable does.

How do we decide that?

Reply:
See my comments: https://github.com/stardust-ui/react/pull/1069#discussion_r274023858 and https://github.com/stardust-ui/react/pull/1069#discussion_r274025046